src/qt/guiconstants.h

```diff
@@ -53,4 +55,20 @@ static const int MAX_URI_LENGTH = 255;
 #define QAPP_APP_NAME_DEFAULT "Dogecoin-Qt"
 #define QAPP_APP_NAME_TESTNET "Dogecoin-Qt-testnet"
 
+/* One gigabyte (GB) in bytes */
+static constexpr uint64_t GB_BYTES{1000000000};
+
+/**
+ * Convert configured prune target MiB to displayed GB. Round up to avoid underestimating max disk usage.
+ */
+constexpr inline int PruneMiBtoGB(uint64_t mib) { return (mib + GB_BYTES - 1) / GB_BYTES; }
+
+/**
+ * Convert displayed prune target GB to configured MiB. Round down so roundtrip GB -> MiB -> GB conversion is stable.
+ */
+constexpr inline int64_t PruneGBtoMiB(int gb) { return gb * GB_BYTES / 1024 / 1024; }
+
+// Default prune target displayed in GUI.
+static constexpr int DEFAULT_PRUNE_TARGET_GB{PruneMiBtoGB(MIN_DISK_SPACE_FOR_BLOCK_FILES)};
```

Comment:
I think you missed the +1 here, otherwise awesome ❤️

Reply:
Fixed!